src/screens/Register.jsx

```diff
@@ -20,21 +22,39 @@ const Register = () => {
     setIsLoading(false);
   };
 
-  const assignUserRole = async (email, values) => {
-    const account = await assignRole({ email, ...values });
-    console.log(account);
+  const assignUserRole = async (email, values) => { // I'm not a big fan of how I've handled this part to be honest
+    const { technologies, ...rest } = values
+    if (role === 'candidate')
+      assignRole(role, { email, [role]: {email, ...rest}, technologies: technologies.map(item => item.value) })
```

Comment:
Instead of doing the if here, I would just pass technologies in both cases, the server doesn't care. 

And if you dont want to pass technologies, I would build my object before calling `assignRole()`

`
const company = {  [role]: {email, ...rest} }
const candidate = {  [role]: {email, ...rest}, technologies }
`

And then you just pass it to `assignRole()` based on the role.